setup.py

```diff
@@ -17,13 +17,14 @@
     ),
     package_dir={"": "src"},
     install_requires=[
-        'pandas>=1.2.1',
+        'pandas==1.2.1',
```

Comment:
Why do we need that ?

Reply:
Yes, well, but we cannot be stuck with a Pandas version that seem a bit old. Let's take more recent versions, but without imposing one.